bleak/backends/bluezdbus/client.py

```diff
@@ -587,17 +586,16 @@ async def get_services(
         if not self.is_connected:
             raise BleakError("Not connected")
 
-        if self._services_resolved:
-            return self.services
+        if self._services:
```

Comment:
Lets explicitly check `is not None` here (in case we ever add `__bool__` or otherwise change `BleakGATTServiceCollection` where an empty collection could be evaluated as `False`).